mitosheet/src/mito/components/endo/celleditor/cellEditorUtils.tsx

```diff
@@ -9,26 +9,33 @@ import { getCellDataFromCellIndexes } from "../utils";
 import { CELL_EDITOR_DEFAULT_WIDTH, CELL_EDITOR_MAX_WIDTH } from "./CellEditor";
 
 
-export const getSelectionFormulaString = (selections: MitoSelection[], sheetData: SheetData): string => {
+export const getSelectionFormulaString = (selections: MitoSelection[], sheetData: SheetData, sheetIndex: number): string => {
```

Comment:
This interface still confuses me. Is this the sheetIndex of the selectedSheet? Or the sheetIndex of the edited sheet? Can we rename to be more clear?